src/lib/components/filters/quickFilters.svelte

```diff
@@ -58,7 +58,6 @@
                                 $columns,
                                 analyticsSource
                             );
-                            subSheets = subSheets;
```

Comment:
does this removal of reassignment affect ui changes due to missed reactivity now?